README.md

```diff
@@ -43,7 +60,6 @@ steps:
       notification_title: "{workflow} has {status_message}"
       message_format: "{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}>"
       footer: "Linked Repo <{repo_url}|{repo}> | <{workflow_url}|View Workflow>"
-      notify_when: "failure"
```

Comment:
I noticed that the screenshot shows a success message while `notify_when` is set to only failures. This just makes the config match the screenshot :-)